🧑‍🌾 ament_mypy test_mypy failures in windows repeated CI

Reference build:
* https://ci.ros2.org/job/nightly_win_rep/3656/

Test regressions:
* [ament_mypy.test.test_mypy.test_mypy[1-3]](https://ci.ros2.org/job/nightly_win_rep/3656/testReport/junit/ament_mypy.test/test_mypy/test_mypy_1_3_/)
* [ament_mypy.test.test_mypy.test_mypy[2-3]](https://ci.ros2.org/job/nightly_win_rep/3656/testReport/junit/ament_mypy.test/test_mypy/test_mypy_2_3_/)
* [ament_mypy.test.test_mypy.test_mypy[3-3]](https://ci.ros2.org/job/nightly_win_rep/3656/testReport/junit/ament_mypy.test/test_mypy/test_mypy_3_3_/)

These test have been failing since January 25th


<details>
<summary>
Log output:
</summary>

```
_______________________________ test_mypy[1-3] ________________________________
test\test_mypy.py:21: in test_mypy
    assert rc == 0, 'Found code style errors / warnings'
E   AssertionError: Found code style errors / warnings
E   assert 1 == 0
---------------------------- Captured stdout call -----------------------------

8 files checked
2 errors
ament_mypy\main.py: note: In function "_get_files":
ament_mypy\main.py:233:45: error: "str" has no attribute "removesuffix"
Found 1 error in 1 file (checked 8 source files)

_______________________________ test_mypy[2-3] ________________________________
test\test_mypy.py:21: in test_mypy
    assert rc == 0, 'Found code style errors / warnings'
E   AssertionError: Found code style errors / warnings
E   assert 1 == 0
---------------------------- Captured stdout call -----------------------------

8 files checked
2 errors
ament_mypy\main.py: note: In function "_get_files":
ament_mypy\main.py:233:45: error: "str" has no attribute "removesuffix"
Found 1 error in 1 file (checked 8 source files)

_______________________________ test_mypy[3-3] ________________________________
test\test_mypy.py:21: in test_mypy
    assert rc == 0, 'Found code style errors / warnings'
E   AssertionError: Found code style errors / warnings
E   assert 1 == 0
---------------------------- Captured stdout call -----------------------------
8 files checked
2 errors
ament_mypy\main.py: note: In function "_get_files":
ament_mypy\main.py:233:45: error: "str" has no attribute "removesuffix"
Found 1 error in 1 file (checked 8 source files)

```

</details>

--- a/ament_mypy/ament_mypy/main.py
+++ b/ament_mypy/ament_mypy/main.py
@@ -230,7 +230,8 @@ def _get_files(paths: List[str]) -> List[str]:
                         type_stub_path = os.path.join(dirpath, filename)
                         files.append(type_stub_path)
 
-                        regular_file_path = type_stub_path.removesuffix('i')
+                        # removes suffix i
+                        regular_file_path = type_stub_path[:-len('i')]
                         # Use type stub over file
                         if regular_file_path in files:
                             files.remove(regular_file_path)

